source/extensions/common/wasm/context.cc

```diff
@@ -1132,7 +1138,7 @@ WasmResult Context::setProperty(absl::string_view path, absl::string_view value)
 }
 
 WasmResult Context::declareProperty(absl::string_view path,
-                                    std::unique_ptr<const WasmStatePrototype> state_prototype) {
+                                    std::unique_ptr<const CelStatePrototype> state_prototype) {
```

Comment:
style: define `CelStatePrototypeConstPtr`?

Reply:
Seems like this is the only place it would be used. Not sure if it's worth it.

Reply:
not strong opinion but it is used as map value type as well, so it will be clearer.

Reply:
done